apis/kueue/v1beta1/workload_conversion_test.go

```diff
@@ -0,0 +1,323 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	"sigs.k8s.io/kueue/apis/kueue/v1beta2"
+)
+
+func TestWorkloadConvertTo(t *testing.T) {
+	testCases := map[string]struct {
+		input    *Workload
+		expected *v1beta2.Workload
+	}{
+		"nil AccumulatedPastExexcutionTimeSeconds": {
+			input: &Workload{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-workload",
+					Namespace: "default",
+				},
```

Comment:
Let's extract `defaultObjectMeta` to avoid duplicating, it is not different between objects.
It can be scoped in test.

Reply:
Fixed!